api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/payments/PaymentService.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.dhsc.htbhf.claimant.service.payments;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.dhsc.htbhf.claimant.entity.PaymentCycle;
+import uk.gov.dhsc.htbhf.claimant.message.MessagePayloadFactory;
+import uk.gov.dhsc.htbhf.claimant.message.MessageQueueDAO;
+import uk.gov.dhsc.htbhf.claimant.message.MessageType;
+import uk.gov.dhsc.htbhf.claimant.message.payload.MessagePayload;
+
+@Service
+@AllArgsConstructor
+@Slf4j
+public class PaymentService {
+
+    private MessageQueueDAO messageQueueDAO;
```

Comment:
I think this should be injected by the interface `MessageQueueClient` then when/if we decide to swap implementation to RabbitMQ, this code will not need to change. Also, the code to use it below looks a bit odd as we're calling `sendMessage` on a DAO.

Reply:
Updated to use the interface.